packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -37,58 +44,317 @@ import {Heading, Text} from '@react-spectrum/typography';
 
 ```tsx example
 <DialogTrigger>
-  <ActionButton>Trigger</ActionButton>
+  <ActionButton>Check connectivity</ActionButton>
   {(close) => (
     <Dialog>
-      <Header><Heading>The Heading</Heading></Header>
+      <Heading>Internet Speed Test</Heading>
+      <Header>
+        <StatusLight variant="positive">
+          Connection status
+        </StatusLight>
+      </Header>
       <Divider />
       <Content>
-        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sit amet tristique risus. In sit amet suscipit lorem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In condimentum imperdiet metus non condimentum. Duis eu velit et quam accumsan tempus at id velit. Duis elementum elementum purus, id tempus mauris posuere a. Nunc vestibulum sapien pellentesque lectus commodo ornare.</Text>
+        <Text>
+          Start speed test?
+        </Text>
       </Content>
-      <Footer>
+      <ButtonGroup>
         <Button variant="secondary" onPress={close}>Cancel</Button>
         <Button variant="cta" onPress={close}>Confirm</Button>
-      </Footer>
+      </ButtonGroup>
     </Dialog>
   )}
 </DialogTrigger>
 ```
 
 ## Content
```

Comment:
@devongovett IMO, popover Dialogs and other types of dialogs should be covered in the DialogTrigger docs page since that is where the `type` prop will be shown in the prop table and where we can best explain the other dialog types and when to use them